operations/enterprises.md

```diff
@@ -297,163 +300,295 @@ Returns all spaces of an enterprise associated with the connector integration.
+
+#### Resource data update
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Discriminator` | string [Resource data discriminator](#resource-data-discriminator) | required | Defines the type of the resource. |
+| `Value` | object | required | Based on Resource data discriminator, e.g. [Space resource data update](#space-resource-data-update) |
+
+#### Space resource data update
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `FloorNumber` | [String update value](reservations.md#string-update-value) | required | New number of the floor the space is on. |
+
+### Response
+
+```javascript
+{}
+```
+
+#### Resource category
 
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
 | `Id` | string | required | Unique identifier of the category. |
-| `IsActive` | bool | required | Whether the space category is still active. |
-| `Name` | string | required | Name of the category. |
+| `IsActive` | bool | required | Whether the category is still active. |
+| `Type` | string [Resource category type](#resource-category-type) | required | Type of the category. |
 | `Names` | [Localized text](enterprises.md#localized-text) | required | All translations of the name. |
-| `ShortName` | string | optional | Short name \(e.g. code\) of the category. |
 | `ShortNames` | [Localized text](enterprises.md#localized-text) | required | All translations of the short name. |
-| `Description` | string | optional | Description of the category. |
 | `Descriptions` | [Localized text](enterprises.md#localized-text) | required | All translations of the description. |
 | `Ordering` | number | required | Ordering of the category, lower number corresponds to lower category \(note that uniqueness nor continuous sequence is guaranteed\). |
-| `UnitCount` | number | required | Count of units that can be accommodated \(e.g. bed count\). |
-| `ExtraUnitCount` | number | required | Count of extra units that can be accommodated \(e.g. extra bed count\). |
-| `ImageIds` | array of string | required | Unique identifiers of the space category images. |
+| `Capacity` | number | required | Capacity that can be accommodated \(e.g. bed count\). |
+| `ExtraCapacity` | number | required | Extra capacity that can be accommodated \(e.g. extra bed count\). |
+
+#### Resource category type
+
+* `Room`
+* `Dorm`
+* `Bed`
+* ...
+
+#### Resource category assignment
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ResourceId` | string | required | Unique identifier of the resource. |
+| `CategoryId` | string | required | Unique identifier of the category that resource is assigned to. |
+
+#### Resource category image assignment
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `CategoryId` | string | required | Unique identifier of the category that image is assigned to. |
+| `ImageId` | string | required | Unique identifier of the image. |
```

Comment:
Inconsistent ordering of properties. Either by alphabet or same logical way (first item being assigned, second item being assigned to).

Reply:
I'd avoid alphabetical.